site/content/events/2016-seattle/index.html

```diff
@@ -11,8 +11,7 @@
 <center>
   <table><tr><td>
         <center>
-        <!-- img border=0 width="160" height="240" src="./yourlogohere.png">
--->
+        <!-- <img border=0 width="160" height="240" src="./yourlogohere.png"> -->
```

Comment:
I think you want this to be <img width="160" height="240" src="./Seattle_Skyline_tiny.jpg"> ?